pmm-app/src/pmm-settings/components/SignUp/SignUp.tsx

```diff
@@ -0,0 +1,126 @@
+import React, {
+  FC,
+  useState
+} from 'react';
+import { Form as FinalForm, Field as FinalField, FormRenderProps } from 'react-final-form';
```

Comment:
I would probably not rename the exports unless there is a clash between those and some other names.

Reply:
You're right! There was a clash before, but not anymore :) Done